infra/resources/prod/main.tf

```
@@ -69,4 +42,12 @@ module "commons" {
   snet_pendpoints_address_space   = ["10.30.2.0/23"]
   cidr_subnet_fnsubscription      = ["10.30.5.0/24"]
   cidr_subnet_fnsubscriptionasync = ["10.30.6.0/24"]
+
+  apim_subnet_cidr = ["10.30.8.0/24"]
```

nonblocking: to follow the naming of the other variables containing cidrs
```suggestion
  cidr_subnet_apim = ["10.30.8.0/24"]
```